test2017Data.js

```diff
@@ -3,7 +3,7 @@ export default {
   "type": "staff-picks",
   "date": "2017-01",
   "title": "Winter 2016 Staff Picks",
-  "heroImageUrl": null,
+  "heroImageUrl": 'http://staff-picks-dev.us-east-1.elasticbeanstalk.com/books-music-dvds/recommendations/staff-picks/src/client/images/desktop.banner.YA.FIN.png',
```

Comment:
This is a self-referencing image url! I know it'll get updated eventually, just wanted to point it out.